src/axom/quest/tests/CMakeLists.txt

```diff
@@ -136,8 +136,6 @@ if (ENABLE_MPI AND AXOM_ENABLE_SIDRE)
 
     set(quest_regression_depends
         ${quest_tests_depends}
-        conduit::conduit
-        sidre
```

Comment:
Were these libraries simply not necessary?

Reply:
Yes, dependencies are transitive, and CMake handles this for us internally now that we're using the `blt_import_target` macro for our TPLs.

Our policy is to only explicitly depend on a target when we're directly using it. 
E.g. if we had a quest test that had explicit conduit calls, then that test should depend on the `conduit` target.

Reply:
OK, that makes sense.